src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -65,7 +80,7 @@ public JsonAdaptedPerson(Person source) {
         phone = source.getPhone().value;
         email = source.getEmail().value;
         address = source.getAddress().value;
-        tag = source.getTag().getTagName();
+        tag = source.getTag().getTagShort();
```

Comment:
Great spot ! I like how you were able to debug the issue of the json storing the name of the tag rather than the enum. LGTM 👍🏾